x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_map.tsx

```diff
@@ -27,12 +29,18 @@ export const LocationMap = ({ monitorLocations }: LocationMapProps) => {
   const upPoints: LocationPoint[] = [];
   const downPoints: LocationPoint[] = [];
 
+  let isGeoInfoMissing = false;
+
   if (monitorLocations?.locations) {
     monitorLocations.locations.forEach((item: any) => {
-      if (item.summary.down === 0) {
```

Comment:
We should add some tests for this functionality, it's pretty nested.

Reply:
i have added tests.

Reply:
have updated the warning message format 
![image](https://user-images.githubusercontent.com/3505601/72355122-21c31600-36e7-11ea-8d57-b0d2bd8e3c0c.png)